test/Hydration-test.js

```diff
@@ -77,6 +77,14 @@ describe('HydrationLog', function() {
     expect(userOneHydrationLogLog).to.be.an.instanceOf(HydrationLog);
   });
 
+  it.skip('should store user id', function() {
```

Comment:
I'm thinking we should eliminate `userID` key value when mapping raw data to `log` property of `HydrationLog` (`HydrationLog.log` would like like this: 

`[{date: x, numOunces: x}, {date: x, numOunces: x}, {date: x, numOunces: x}]`)

BUT we should store `userID` itself as property of `HydrationLog` to make it super easy to access data from user class